src/main/webapp/app/course/tutorial-groups/shared/tutorial-group-detail/tutorial-group-detail.component.html

```diff
@@ -137,7 +137,7 @@ <h3>{{ 'artemisApp.pages.tutorialGroupDetail.title' | artemisTranslate }}:</h3>
         <dd>
             <ng-container *ngIf="sessions && sessions.length > 0; else sessionsNotFound">
                 <div class="scrollbar table-wrapper-scroll-y border border-lightgrey p-3">
-                    <jhi-tutorial-group-sessions-table [timeZone]="timeZone" [sessions]="sessions" [tutorialGroup]="tutorialGroup" [showIdColumn]="false">
+                    <jhi-tutorial-group-sessions-table [timeZone]="timeZone" [sessions]="sessions" [tutorialGroup]="tutorialGroup" [showIdColumn]="false" [isReadOnly]="true">
```

Comment:
Is it intentional to set isReadOnly to always true?

Reply:
Yes it should not be editable on the detail page